### ISSUE TITLE
💻 Add tick and cross symbols to Puzzle adventures

### ISSUE THREAD

**Issue by juliabolt** [hedyorg/hedy#5069]
**Idea incl level**

In general, but specifically in the Puzzle adventures (multiple levels) it would be helpful to have another indication than just the colors red and green to see what is correct and what is not. In my opinion, a good option would be to add a symbol like a tick mark and a cross, or something similar. 

**Additional context**
Add any other context about the idea here.

I would like this change because my fiancé is colorblind and this is what he sees compared to what I see:

![WhatsApp Image 2024-02-05 at 18 36 49_ae54a9e1](https://github.com/hedyorg/hedy/assets/32330467/016c28c2-c158-46e7-bd73-70780975a42a)

He cannot see the difference between the red and green, and there are more people who will have this issue!

**Comment by Felienne** [comment]
Maybe ask @gisellandrade for ideas for the icons?
